docs/low-code-spark/gems/source-target/warehouse/bigquery.md

```diff
@@ -0,0 +1,179 @@
+---
+title: BigQuery
+id: bigquery
+description: BigQuery
+sidebar_position: 6
+tags:
+  - gems
+  - warehouse
+  - bigquery
+---
+
+:::info Built on
+This connector is built on top of the already available [`spark-bigquery-connector`](https://github.com/GoogleCloudDataproc/spark-bigquery-connector/)
+
+To use this Gem in Prophecy, `com.google.cloud.spark:spark-3.3-bigquery-pushdown_2.12:0.30.0` Maven external dependency needs to be installed on cluster if it's not present.
+For installing dependencies from Prophecy UI, Please check [dependency management docs](https://docs.prophecy.io/low-code-spark/extensibility/dependencies)
```

Comment:
Rephrase it to for non-databricks clusters we need to install the corresponding library and add the link to the library compatiblity matrix in the docs